packages/react-charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -145,8 +159,8 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
           </FocusZone>
           <FocusZone direction={FocusZoneDirection.horizontal}>
             <div>
-              <svg className={this._classNames.chart} aria-label={data?.chartTitle}>
-                <g>{bars[0]}</g>
+              <svg ref={this.barChartSvgRef} className={this._classNames.chart} aria-label={data?.chartTitle}>
```

Comment:
In dark mode the outline seems to be more than 2px and getting cut on the right side. Can you check

Reply:
same for multi stacked

Reply:
The behaviour of focus indicator has this thing, where it consists of a black and a white outline enclosed on the top of it making it look a lot bigger in dark mode, let's discuss this in tommorrow call, what would be the best way forward

Reply:
![image](https://github.com/microsoft/fluentui/assets/70366079/94b9d782-55bd-4de0-895d-af5af3cea01d)

Reply:
Made the margin 3 px, and reverted back the borders to black and white outline

![image](https://github.com/microsoft/fluentui/assets/70366079/652cd86a-8c5f-4571-bca6-9358413a2def)